internal/mount/mount.go

```diff
@@ -23,4 +23,6 @@ type Options struct {
 	// Allows the mounting over a non-empty directory. The files in it will be shadowed by the freshly created mount.
 	// Supported only on Fuse.
 	FuseAllowNonEmptyMount bool
+	// Use WebDav even on platforms that support FUSE.
+	PreferWebDav bool
```

Comment:
nit: prefer `preferWebDAV` in golang

Reply:
Changed to `PreferWebDAV`. I left it as an exported name because it's set from `cli/command_mount.go`.